ci/do_ci.sh

```diff
@@ -300,6 +300,9 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
   echo "Building and testing ${TEST_TARGETS[*]}"
   bazel_with_collection test "${BAZEL_BUILD_OPTIONS[@]}" "${COMPILE_TIME_OPTIONS[@]}" -c dbg "${TEST_TARGETS[@]}" --test_tag_filters=-nofips --build_tests_only
 
+  echo "Building and testing with wasm=wasmtime: ${TEST_TARGETS[*]}"
+  bazel_with_collection test --define wasm=wasmtime "${COMPILE_TIME_OPTIONS[@]:2}" "${COMPILE_TIME_OPTIONS[@]}" -c dbg "${TEST_TARGETS[@]}" --test_tag_filters=-nofips --build_tests_only
```

Comment:
You use `${COMPILE_TIME_OPTIONS}` twice, instead of `${BAZEL_BUILD_OPTIONS}`.

Also, I think the `--define wasm=wavm` is going to override `--define wasm=wasmtime` here. I'd prefer if you could remove `--define wasm=wavm` from `${COMPILE_TIME_OPTIONS}` and add `--define wasm=wavm` and `--define wasm=wasmtime` explicitly to each invocation to make it clearer.

Reply:
thanks done in https://github.com/envoyproxy/envoy/pull/13932/commits/e4fa56ef895fff183abf84daf53a6d91227041bd